bin/server.js

```diff
@@ -22,4 +25,18 @@ if (config.util.getEnv('NODE_ENV') === 'development') {
   }
 }
 
-require('core/server/base').runServer();
+require('core/server/base').runServer().then(() => {
+  if (config.get('isDevelopment')) {
+    const basePath = config.get('basePath');
+    const watcher = chokidar.watch(path.join(basePath, 'src'));
+
+    watcher.on('ready', () => {
+      console.log('👀 The file watcher for server hot reload is ready.');
+
+      watcher.on('all', () => {
+        // Touching this file triggers a server reload.
```

Comment:
This is pretty non-intuitive (I have no idea how it works). Could you elaborate on this comment? Or is it magical to you, too?

Reply:
We require `piping` a few lines above, I don't really know why this is there but it is super useful to reload a node server obviously: https://github.com/mdlawson/piping.